java/src/main/java/com/codurance/training/tasks/DeleteImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.codurance.training.tasks;
+
+import java.io.PrintWriter;
+import java.util.Iterator;
+import java.util.List;
+
+import static com.codurance.training.tasks.TaskList.tasks;
+
+public class DeleteImpl implements AddService{
+    @Override
+    public void CRUD(String id, PrintWriter out) {
```

Comment:
Logically there is no relation between AddService & DeleteImpl.... better to create seperate interface for delete